collectors/prometheus.go

```diff
@@ -106,7 +104,9 @@ func NewPrometheusExporter(cfg *PrometheusConfig, lnd *lndclient.LndServices,
 		NewInfoCollector(lnd.Client, errChan),
 	}
 
+	var htlcMonitor *htlcMonitor
```

Comment:
We can't just leave this `nil`, otherwise this will cause a panic when we call `start()` here: https://github.com/lightninglabs/lndmon/blob/662470e3e20fc11912c426d2c5d3a6978a60f109/collectors/prometheus.go#L153

I thin we just have to put `if !monitoringCfg.DisableHtlc {` around the start and stop of `p.htlcMonitor` instead.

Reply:
Makes sense. Pushed an updated version.